src/heuristics/mod.rs

```diff
@@ -1,4 +1,8 @@
 //! (Meta)Heuristic templates.
+//!
+//! The templates in this module allow customization of certain operators
+//! similar to other frameworks (e.g. a custom mutation operator) and are meant to
+//! provide a starting point for more specialized configurations.
```

Comment:
Maybe we could add a note that it is also perfectly valid to just copy-paste a template and modify it to one's liking.

Reply:
I added a sentence.